syntax = "proto3";



message HelloRequest{
    string name = 1;
}

message HelloResponse{
    string message = 1;
}

message UserInfoRequest{
    string email = 1;
}

message UserInfoResponse{
    string first_name = 1;
    string last_name = 2;
    string country = 3;
    string phone = 4;
}

service HelloWorldService{
    rpc SayHello(HelloRequest) returns(HelloResponse);
    rpc GetUserInfo(UserInfoRequest) returns(UserInfoResponse);
}